### ISSUE TITLE
Migration Generator support for single quoted decorators

### ISSUE THREAD

**Issue by tsanchev** [yiisoft/yii2#12736]
### What steps will reproduce the problem?

I suppose this is not possible for the moment but will be good to have it

`yii migrate/create create_brands_table  --fields="logo:'LONGBLOB'"`
### What is the expected result?

```
public function up()
{
    $this->createTable('brands', [
        'id' => $this->primaryKey(),
        'logo' => 'LONGBLOB',
    ]);
}
```
### What do you get instead?

```
public function up()
{
    $this->createTable('brands', [
        'id' => $this->primaryKey(),
        'logo' => $this->'LONGBLOB'(),
    ]);
}
```
### Additional info

| Q | A |
| --- | --- |
| Yii version | 2.0.10-dev |
| PHP version | 5.6.9 |

**Comment by SilverFire** [comment]
@mdmunir

**Comment by samdark** [comment]
Won't be implemented in 2.0.
